From 7a0bac9436e69b30f5b3aaadf8d2ad87d8d04559 Mon Sep 17 00:00:00 2001
From: Samin Guo <samin.guo@starfivetech.com>
Date: Fri, 3 Mar 2023 16:59:25 +0800
Subject: [PATCH 58/70] riscv: dts: starfive: jh7110: Add syscon to support phy
 interface settings

The phy interface needs to be set in syscon, the format is as follows:
starfive,syscon: <&syscon, offset, mask>

Signed-off-by: Samin Guo <samin.guo@starfivetech.com>
---
 arch/riscv/boot/dts/starfive/jh7110.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/riscv/boot/dts/starfive/jh7110.dtsi b/arch/riscv/boot/dts/starfive/jh7110.dtsi
index b0e87198808a..8b73d01c226c 100644
--- a/arch/riscv/boot/dts/starfive/jh7110.dtsi
+++ b/arch/riscv/boot/dts/starfive/jh7110.dtsi
@@ -694,6 +694,7 @@ gmac0: ethernet@16030000 {
                        snps,en-tx-lpi-clockgating;
                        snps,txpbl = <16>;
                        snps,rxpbl = <16>;
+		       starfive,syscon = <&aon_syscon 0xc 0x1c0000>;
                        status = "disabled";
                        phy-handle = <&phy0>;
 
@@ -736,6 +737,7 @@ gmac1: ethernet@16040000 {
                        snps,en-tx-lpi-clockgating;
                        snps,txpbl = <16>;
                        snps,rxpbl = <16>;
+		       starfive,syscon = <&sys_syscon 0x90 0x1c>;
                        status = "disabled";
                        phy-handle = <&phy1>;
 
-- 
2.39.2

